### PR TITLE
Updating pipeline: Master-ImportAccUsersComtFinResvToDataMart

### PR DESCRIPTION
**Pull request by sarmaevanidfe**

### CODE REVIEW

File: pipeline/Master-ImportAccUsersComtFinResvToDataMart.json [hunk 1]
```diff
@@ -388,6 +388,33 @@
...
+				"userProperties": [],
+				"typeProperties": {
+					"pipeline": {
+						"referenceName": "ImportAssessorToDM",
+						"type": "PipelineReference"
+					},
+					"waitOnCompletion": true,
+					"parameters": {
+						"RunId": {
+							"value": "@{activity('GetRunId').output.firstRow.RunId}",
```
Comment (uhimabindu) [inline pipeline/Master-ImportAccUsersComtFinResvToDataMart.json:413]: Can we also give values for MaxConcurrentConnections and DIU

**Review by uhimabindu — CHANGES_REQUESTED** [review]

File: pipeline/Master-ImportAccUsersComtFinResvToDataMart.json [hunk 2]
```diff
@@ -388,6 +388,33 @@
 						"DIU": "4"
 					}
 				}
+			},
+			{
+				"name": "ExecuteAssessorEPAO",
```
[the remark this answers is not in the document]
Comment (sarmaevanidfe) [reply pipeline/Master-ImportAccUsersComtFinResvToDataMart.json]: Amended naming convention as per others

File: pipeline/Master-ImportAccUsersComtFinResvToDataMart.json (hunk 1 above)
Comment (sarmaevanidfe) [reply pipeline/Master-ImportAccUsersComtFinResvToDataMart.json:413]: Amended values for MaxConcurrentConnections and DIU

**Review by uhimabindu — APPROVED** [review]
Approved

**Pull request merged**